packages/DatePicker/src/DatePicker.js

```diff
@@ -215,6 +215,7 @@ function DatePicker(props) {
 
   return (
     <Popover
+      style={{ marginTop: "150px" }}
```

Comment:
Don't forget to remove this... I'd suggest adding something like it to `DatePicker.stories.js`